command/loadbot/params.go

```diff
@@ -58,24 +62,21 @@ type loadbotParams struct {
 	gasPrice         *big.Int
 	gasLimit         *big.Int
 	contractArtifact *generator.ContractArtifact
+	constructorArgs  []byte
 }
 
 func (p *loadbotParams) validateFlags() error {
-	// Validate the correct mode type
-	convMode := Mode(strings.ToLower(p.modeRaw))
-	if convMode != transfer && convMode != deploy {
+	// Set and validate the correct mode type
+	p.mode = Mode(strings.ToLower(p.modeRaw))
+	if p.mode != transfer && p.mode != deploy && p.mode != erc20 && p.mode != erc721 {
```

Comment:
This can be a separate function `isValidMode`

Reply:
Added isValidMode method a9000c81b7a864ea173e53de01191f6f649f4496